templates/slide.html

```diff
@@ -420,23 +434,45 @@ <h2 class="popover__title">{{disp_str[i]}}</h2>
  function onPrevPage() {
    // empty previous child elements of textLayer
    $("#textLayerid").empty();
-
+   course = "{{course_name}}";
+    lec = "{{lec_name}}";
    if (pageNum <= 1) {
      return;
    }
+   
+    
    pageNum--;
+   new_slide_name = course.concat("---").concat(lec).concat("---slide").concat((pageNum-1).toString());
+   logdata = JSON.stringify({
+        action: 'prev',
+        middle: new_slide_name, 
```

Comment:
can you check if this way of assignment to new_slide_name works on Line 448 ? middle: "{{course_name}}".concat("---").concat("{{lec_name}}").concat("---slide").concat((pageNum-1).toString());

Reply:
Yeap it worked! So I set the new_slide_name to "{{course_name}}".concat("---").concat("{{lec_name}}").concat("---slide").concat((pageNum-1).toString()); in the onPrevPage function